spec/features/admin/admin_index_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe "admin index page", type: :feature do
+
+  before :each do
+    @user = User.create(full_name:"Chelsea May",
+                            email:"chelsmay@gmail.com",
+                         password:"password",
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space missing after colon.